packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -100,7 +100,7 @@ export interface TokenRatesState extends BaseState {
  * The maximum number of token addresses that should be sent to the Price API in
  * a single request.
  */
-const TOKEN_PRICES_BATCH_SIZE = 100;
+export const TOKEN_PRICES_BATCH_SIZE = 100;
```

Comment:
Anything that's exported from this file is public to the outside, but anything exported from `assetsUtil` is not (just due to how we've defined the exports in `index.ts` for this package). What do you think about moving this to `assetsUtil` and then importing it from here?

Reply:
Definitely!